storage/innobase/handler/ha_innodb.cc

```diff
@@ -23996,3 +24007,59 @@ innodb_encrypt_tables_validate(
 
 	return 0;
 }
+
+static int
+innodb_redo_log_encrypt_validate(
+    /*=================================*/
+    THD *		     thd,  /*!< in: thread handle */
+    struct st_mysql_sys_var *var,  /*!< in: pointer to system
+                                                  variable */
+    void *		     save, /*!< out: immediate result
+                                  for update function */
+    struct st_mysql_value *  value)  /*!< in: incoming string */
+{
+	const char *redo_log_encrypt_input;
+	char	buff[STRING_BUFFER_USUAL_SIZE];
+	int	 len = sizeof(buff);
+
+	ut_a(save != NULL);
+	ut_a(value != NULL);
+
+	redo_log_encrypt_input = value->val_str(value, buff, &len);
+
+	bool legit_value = false;
```

Comment:
Is this new code or copy-pasted from somewhere? If yes, from where?

Reply:
I copied the innodb_encrypt_tables validation and changed it to match redo_log_encrypt.